integration/test_web.py

```diff
@@ -0,0 +1,155 @@
+import sys
+import time
+import shutil
+import json
+import urllib2
+from os import mkdir, unlink, utime
+from os.path import join, exists, getmtime
+
+import util
+
+import requests
+import pytest_twisted
+
+
+def test_index(alice):
+    """
+    we can download the index file
+    """
+    util.web_get(alice._node_dir, "")
```

Comment:
Does `_node_dir` need to be public?  Or maybe `web_get` (or `node_url`?) should be a method of alice.

Reply:
Really that was abusing the process transport anyway, so I refactored this some more (so there's a `TahoeProcess` object now, with a `.transport` and `.node_dir`).